go/adcm/status/handlers.go

```diff
@@ -295,10 +295,67 @@ func showHost(h Hub, w http.ResponseWriter, r *http.Request) {
 		ErrOut4(w, r, "HOST_NOT_FOUND", "unknown host")
 		return
 	}
-	val, _ := h.HostStorage.get(ALL, hostId)
+	val, _ := h.HostStatusStorage.get(ALL, hostId)
 	jsonOut(w, r, val)
 }
 
+func listHost(h Hub, w http.ResponseWriter, r *http.Request) {
+	allow(w, "GET, POST")
+	result, _ := h.ListHostStorage.list()
+	jsonOut(w, r, result)
+
```

Comment:
Is there a reason to have empty lines here?

Reply:
fixed

Reply:
I suggest mention here deletion of previously existed hosts.
And typo in word "Format"

Reply:
fixed